vtex-apps/store-link
# Fix/slugification

#### What problem is this solving?

When using store-link like so:

```json
{
  "link.product#productbrand": {
    "props": {
      "href": "/{brand}"
    },
    "children": [
      "product-brand"
    ]
  }
}
```

The way the slug is created for brands with spaces and characters is not supposed to work as it should be:

https://quickfixes--powerplanet.myvtex.com/refrigeracion-liquida-cooler-master-masterliquid-ml240l-rgb/p?__bindingAddress=b2c.powerplanet.com/es 

This product has the brand "Cooler Master", and when you click on it, it sends you to: 
https://quickfixes--powerplanet.myvtex.com/COOLER%20MASTER?__bindingAddress=b2c.powerplanet.com/es

Which is not a brand page, the brand page is: 
https://quickfixes--powerplanet.myvtex.com/COOLER-MASTER?__bindingAddress=b2c.powerplanet.com/es

#### How to test it?



Check the cooler master link on

https://kiwi--powerplanet.myvtex.com/cooler-cpu-hyper-412s/p?__bindingAddress=b2c.powerplanet.com/es

vs

https://powerplanet.myvtex.com/cooler-cpu-hyper-412s/p?__bindingAddress=b2c.powerplanet.com/es

You will be able to see that the one from the `kiwi` workspace converts the space to a `-`, while the master one uses a space encoded to `%20`.

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/ce280BjADCmsfgps4t/giphy.gif)
